### DISCUSSION TITLE
Possible to fetch other schema in select?

### DISCUSSION THREAD

**Discussion by Eldynn** [supabase/supabase#16898] [Questions] [answered]
I have a link to the storage.objects tables of supabase and I want in a select to fetch the bucket id and name (path) from it, is it possible to do something like, and if yes how?

https://xxxxx.supabase.co/rest/v1/Document?select=some_column,storage.objects(bucket_id,name)

**Comment by GaryAustin1** [comment, marked as answer]
You can not access two schemas at the same time.
You can switch schemas either when you createClient, or with the latest supabase-js with .schema('otherschema') as part of the call.  But everything has to be in that schema.

Edit: you could also use a view or rpc call.

**Reply by Eldynn** [reply]
Even without the supabase client and just with curl request over http it's not possible? Or the limitation is only for the supabase js client ?

**Reply by GaryAustin1** [reply]
A header is used in the api requests to select the one schema used it defaults to public.  So REST API is the limit.
